monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/micro/integrator/delegates/RegistryResourceDelegate.java

```diff
@@ -23,52 +23,59 @@
 import com.google.gson.JsonArray;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
+
 import org.apache.http.HttpEntity;
 import org.apache.http.client.methods.CloseableHttpResponse;
 import org.apache.http.util.EntityUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.wso2.ei.dashboard.core.commons.utils.HttpUtils;
 import org.wso2.ei.dashboard.core.commons.utils.ManagementApiUtils;
-import org.wso2.ei.dashboard.core.db.manager.DatabaseManager;
-import org.wso2.ei.dashboard.core.db.manager.DatabaseManagerFactory;
+import org.wso2.ei.dashboard.core.data.manager.DataManager;
+import org.wso2.ei.dashboard.core.data.manager.DataManagerSingleton;
 import org.wso2.ei.dashboard.core.exception.ManagementApiException;
 import org.wso2.ei.dashboard.core.rest.model.NodeList;
 import org.wso2.ei.dashboard.core.rest.model.RegistryArtifacts;
 import org.wso2.ei.dashboard.core.rest.model.RegistryArtifactsInner;
 import org.wso2.ei.dashboard.core.rest.model.RegistryProperty;
+import org.wso2.ei.dashboard.core.rest.model.RegistryResourceResponse;
 import org.wso2.ei.dashboard.micro.integrator.commons.Utils;
 
 import java.io.IOException;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 /**
  * Delegate class to handle requests from registry resources page.
  */
 public class RegistryResourceDelegate {
     private static final Logger logger = LogManager.getLogger(RegistryResourceDelegate.class);
-    private final DatabaseManager databaseManager = DatabaseManagerFactory.getDbManager();
+    private static final DataManager dataManager = DataManagerSingleton.getDataManager();
+    
+    public List<RegistryProperty> getRgistryFilePoperties(String groupId, String registryPath) 
```

Comment:
Do we use this function anyhwere?

Reply:
No, removed the function and one of its dependant function